libnopegl/src/node_graphicconfig.c

```diff
@@ -266,14 +266,12 @@ static int graphicconfig_init(struct ngl_node *node)
     struct graphicconfig_priv *s = node->priv_data;
     const struct graphicconfig_opts *o = node->opts;
 
-    if (o->stencil_write_mask != -1 &&
-        (o->stencil_write_mask < 0 || o->stencil_write_mask > 0xff)) {
+    if (o->stencil_write_mask != -1 && o->stencil_write_mask > 0xff) {
```

Comment:
This change looks unrelated (same below): the type is still int32 here (probably because there is this set/unset thing in the options)

Reply:
Changed back to int. I will add the proper explict cast in the PR removing `-Wno-sign-conversion`.